automated install/basic-install.sh

```diff
@@ -156,6 +156,9 @@ EOM
 # The runUnattended flag is one example of this
 repair=false
 runUnattended=false
+if [ -z "$PIHOLE_SKIP_FTL_CHECK" ]; then
+    PIHOLE_SKIP_FTL_CHECK=false
+fi
```

Comment:
```suggestion
skipFTL=false
```

Can we keep the scheme the same for all of the undocumented flags? I'm not a fan of two different styles. My preference is to copy the existing style, so use a `--skipftl` instead of environment variables. That will also need to include an additional case at line 167 below `"--unattended"`.